ng -v error reflect-metadata after update cli version beta.20-1

> Please provide us with the following information:
> ---------------------------------------------------------------

### OS?
> ubuntu 14.04

### Versions.
angular-cli@1.0.0-beta.20-1
node 7.0.0
npm 3.10.9


### Repro steps.
update latest cli
```
npm rm -g angular-cli
npm i -g angular-cli
```
then 
```
ng -v
```

have done the same with npm cache clean, same error

### The log given by the failure.
```sh
$ ng -v
Cannot find module 'reflect-metadata'
Error: Cannot find module 'reflect-metadata'
```


### Mention any other details that might be useful.
Also when doing `npm ls -g --depth=0` shows
```sh
npm ERR! peer dep missing: @angular/compiler-cli@^2.1.0, required by @ngtools/webpack@1.1.7
npm ERR! peer dep missing: @angular/core@^2.0.0, required by @ngtools/webpack@1.1.7
npm ERR! peer dep missing: reflect-metadata@^0.1.8, required by @ngtools/webpack@1.1.7
```
didn't happen in previous version.

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

full stack traces
```sh
$ ng -v
Cannot find module 'reflect-metadata'
Error: Cannot find module 'reflect-metadata'
    at Function.Module._resolveFilename (module.js:472:15)
    at Function.Module._load (module.js:420:25)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/antony-budianto/.nvm/versions/node/v7.0.0/lib/node_modules/angular-cli/node_modules/@ngtools/webpack/src/index.js:5:1)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/antony-budianto/.nvm/versions/node/v7.0.0/lib/node_modules/angular-cli/models/webpack-build-typescript.js:5:17)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/antony-budianto/.nvm/versions/node/v7.0.0/lib/node_modules/angular-cli/models/webpack-config.js:2:34)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/antony-budianto/.nvm/versions/node/v7.0.0/lib/node_modules/angular-cli/tasks/build-webpack.js:6:24)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/antony-budianto/.nvm/versions/node/v7.0.0/lib/node_modules/angular-cli/commands/build.js:3:23)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Class.includedCommands (/home/antony-budianto/.nvm/versions/node/v7.0.0/lib/node_modules/angular-cli/addon/index.js:21:16)
    at /home/antony-budianto/.nvm/versions/node/v7.0.0/lib/node_modules/angular-cli/angular-cli/lib/models/project.js:392:61
    at Array.forEach (native)
    at Project.addonCommands (/home/antony-budianto/.nvm/versions/node/v7.0.0/lib/node_modules/angular-cli/angular-cli/lib/models/project.js:391:15)
    at Project.eachAddonCommand (/home/antony-budianto/.nvm/versions/node/v7.0.0/lib/node_modules/angular-cli/angular-cli/lib/models/project.js:426:30)
    at module.exports (/home/antony-budianto/.nvm/versions/node/v7.0.0/lib/node_modules/angular-cli/angular-cli/lib/cli/lookup-command.js:33:13)
    at CLI.<anonymous> (/home/antony-budianto/.nvm/versions/node/v7.0.0/lib/node_modules/angular-cli/angular-cli/lib/cli/cli.js:34:26)
    at tryCatch (/home/antony-budianto/.nvm/versions/node/v7.0.0/lib/node_modules/angular-cli/node_modules/rsvp/dist/rsvp.js:538:12)
    at invokeCallback (/home/antony-budianto/.nvm/versions/node/v7.0.0/lib/node_modules/angular-cli/node_modules/rsvp/dist/rsvp.js:553:13)
    at publish (/home/antony-budianto/.nvm/versions/node/v7.0.0/lib/node_modules/angular-cli/node_modules/rsvp/dist/rsvp.js:521:7)
    at flush (/home/antony-budianto/.nvm/versions/node/v7.0.0/lib/node_modules/angular-cli/node_modules/rsvp/dist/rsvp.js:2373:5)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)

```

--- a/package.json
+++ b/package.json
@@ -34,10 +34,6 @@
     "npm": ">= 3.0.0"
   },
   "author": "Angular Authors",
-  "contributors": [
-    "Rody Haddad <npm@rodyhaddad.com> (http://rodyhaddad.com/)",
-    "Igor Minar <igor@angularjs.org>"
-  ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/angular/angular-cli/issues"

--- a/packages/angular-cli/package.json
+++ b/packages/angular-cli/package.json
@@ -27,6 +27,8 @@
   "dependencies": {
     "@angular-cli/ast-tools": "^1.0.1",
     "@angular-cli/base-href-webpack": "^1.0.0",
+    "@angular/compiler-cli": "^2.1.0",
+    "@angular/core": "^2.1.0",
     "@ngtools/webpack": "latest",
     "angular2-template-loader": "^0.5.0",
     "awesome-typescript-loader": "^2.2.3",
@@ -82,6 +84,7 @@
     "quick-temp": "0.1.5",
     "raw-loader": "^0.5.1",
     "readline2": "0.1.1",
+    "reflect-metadata": "^0.1.8",
     "remap-istanbul": "^0.6.4",
     "resolve": "^1.1.7",
     "rimraf": "^2.5.3",

--- a/packages/webpack/package.json
+++ b/packages/webpack/package.json
@@ -30,7 +30,7 @@
   "peerDependencies": {
     "typescript": "^2.0.2",
     "@angular/compiler-cli": "^2.1.0",
-    "@angular/core": "^2.0.0",
+    "@angular/core": "^2.1.0",
     "reflect-metadata": "^0.1.8"
   }
 }

